utilities/getPageMetadata.ts

```diff
@@ -47,10 +52,10 @@ export const getPageMetadata = ({
   const { siteTitle, siteDescription, siteUrlProd: siteUrl } = config;
 
   const title = seoTitle || pageTitle;
-  const description = seoDescription || siteDescription;
+  const description = seoDescription || dek || siteDescription;
   const ogTitle = og_title || title;
   const ogDescription = og_description || description;
-  const heroImageCropped = getProcessedImage(filename, '1200x630', focus);
+  const heroImageCropped = getProcessedImage(filename || bgFilename, '1200x630', focus || bgFocus);
```

Comment:
If a foreground hero image isn't provided, use the background image if there's one (this is how they've been doing for MVP stories - the background image becomes the hero).